### DISCUSSION TITLE
Realm.getGlobalInstanceCount() java method equivalent

### DISCUSSION THREAD

**Discussion by rudyspano** [realm/realm-dotnet#2780] [Q&A] [answered]
Hello, we use realm-dotnet in a Xamarin Forms App.
We try to use best practices regarding instance disposal but I would like to check if all instances are closed.
Is there a way to call something similar to Realm.getGlobalInstanceCount()  in order to check if all instances are correclty closed ?

Thanks

https://github.com/realm/realm-java/issues/6248

**Comment by papafe** [comment, marked as answer]
Hi @rudyspano, no we don't have anything like that in the .NET SDK.
